### ISSUE TITLE
Missing package.json?

### ISSUE THREAD

**Issue by aviflax** [structurizr/puppeteer#1]
Thanks for creating this repo! It’s super helpful.

One minor problem: the readme doesn’t say how to download the dependencies, and one can’t just run `npm install` (which AFAIK is the usual way to install the deps of a node program) because `package.json` isn’t included in the repo.

So maybe it should be included?

Alternatively, I figured out that I can just run `npm install puppeteer` and that works just fine. So another option might be to remove `package-lock.json` and add some instructions to the README that users should simply run `npm install puppeteer`? I’d be happy to contribute such a change as a pull request if you’d like.

**Comment by simonbrowndotje** [comment]
Thanks, yes, the ```package-lock.json``` file shouldn't have been committed, so I've removed it. Installing Puppeteer is relatively straightforward although, IIRC, the first time I did it I had to install ```npm```, which I think meant installing ```brew``` too, which also meant updating something else first. Since I'd rather not get into debugging Puppeteer installations, I've just added a link to the Puppeteer website. 🙂
